lib/pullrequest-decorator-reporter.js

```diff
@@ -128,8 +128,18 @@ class PullRequestDecoratorReporter {
     }
   }
 
-  done(error, summary) {
+ done(error, summary) {
     if (!this.options.export) {
+
+      let x  = this.buildReport(this.context.assertions, summary.collection.items);
+      console.log(x.items);
+
+      this.githubService.createPullRequestComment(
```

Comment:
return this.githubService.decoratePullRequest(this.buildReport(this.context.assertions, summary.collection.items))
.catch((exception) => {
        console.log(`[+] An error occured : ${exception}`)
})
.finally(console.log("[+] Finished adding pull request comment report"))

Reply:
Done